pos_price_to_weight/data/barcode_rule.xml

```diff
@@ -5,7 +5,7 @@ Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
 License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 -->
 
-<openerp><data>
+<odoo><data>
 
     <record id="rule_price_to_weight" model="barcode.rule">
```

Comment:
Please fix indentation issues. XML files should be indented with `spaces` only or `tabs` only. Not with a mix of them. I prefer `spaces` indentation.

Feel free to fix that when you see a mixing of spaces and tabs in an xml or js file when moving to a new version. :)

Reply:
I only find spaces in this file, no tabs? I found mixtures in other `*.xml` files though, I'll fix those in a separate commit.